Dockerfile-auth

```diff
@@ -43,14 +45,15 @@ RUN if [ "${DOCKER_FAKE_RELEASE}" = "YES" ]; then \
 
 # simplify repeated -C calls with SUBDIRS?
 RUN mkdir /build && \
+    LUAVER=$([ -z "${NO_LUA_JIT##*$(uname -m)*}" ] && echo 'lua5.3' || echo 'luajit') && \
```

Comment:
`uname -m` is bad when cross compiling. Might not be relevant in docker, but before anyone copies this elsewhere.

Also, if you can ensure luajit is not installed (Build-Conflicts), you don't need LUAVER at all.

Reply:
> `uname -m` is bad when cross compiling. Might not be relevant in docker, but before anyone copies this elsewhere.

Ah,  we should reuse the `$(dpkg-architecture -q DEB_BUILD_GNU_TYPE)` (perhaps a different value) for that then.

Reply:
@zeha Switch `uname -m` to something like `dpkg --print-architecture` is fine. I am not sure what `dpkg --print-architecture` produces on `arm64`. That might be worth checking out so I can update my PR. @Habbie Can you confirm what `dpkg --print-architecture` on `arm64` produces?

Just an FYI `dpkg --print-architecture` produces `ppc64el` not `ppc64le`. It's not an issue for this statement as `ppc64le` has a Lua JIT. 

Another thing is `dpkg` works fine on `debian`/`ubuntu` based systems, but installing `dpkg` on other distros like `centos` or `ubi` do not make sense. Is PDNS going to expand their docker image offerings? Or just keep it to `debian` for now? I did see some interest in `alpine`.

Reply:
```
peter@buildbot-worker-arm64-1:~$ dpkg --print-architecture
arm64
```

Reply:
We have no plans to support anything other than Debian for our Docker images.

Reply:
Just replaced `uname -m` with `dpkg --print-architecture`. With that I had to modify the `NO_LUA_JIT` variable.